src/components/SideBar.js

```diff
@@ -91,8 +97,30 @@ Menu.propTypes = {
   title: PropTypes.string,
 };
 
+const fetchAnnouncements = async () => {
+  const res = await fetch(process.env.NEXT_PUBLIC_NEW_ANNOUNCEMENTS_URL);
+  const { data } = await res.json();
+  return data;
+};
```

Comment:
I would prefer to use something like react-query in our codebase instead of directly calling endpoints with fetch https://react-query.tanstack.com/

Reply:
I can help set this up on Monday if you want.

Reply:
Good point. Does react-query also make it easier to mock the requests/responses in tests?

Reply:
It wraps fetch with a hook so I don't think it will make it easier

Reply:
Okay, I found a comment by the author about this and his solution seems sufficient I think https://github.com/tannerlinsley/react-query/issues/113#issuecomment-586822007